build.gradle

```diff
@@ -22,3 +26,8 @@ wrapper {
   gradleVersion = '3.2.1'
   distributionType = Wrapper.DistributionType.ALL
 }
+
+coveralls {
```

Comment:
why is this at the top level `build.gradle` when it's configured specifically for only one submodule?